docs/src/user-guide/url-parameters.md

```
@@ -0,0 +1,64 @@
+# URL Parameters
+
+The YScope Log Viewer supports various URL parameters to control its behavior and state. The 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)